Pod/Classes/ViewControllers/ARMapContainerViewController.m

```diff
@@ -154,13 +154,18 @@ - (void)userSuppliedLocation:(CLLocation *)userLocation
 
 - (void)showCityPicker
 {
+    if (self.cityPickerContainerView) {
```

Comment:
cityPickerContainerView is set to `nil` here
https://github.com/artsy/emission/pull/1473/files#diff-a6663cfe9d0e55272a71b62f28e85090R219